packages/gatsby/src/commands/build.js

```diff
@@ -2,7 +2,7 @@
 
 const path = require(`path`)
 const report = require(`gatsby-cli/lib/reporter`)
-const buildHTML = require(`./build-html`)
+import buildHTML from "./build-html"
```

Comment:
As per above comment, this could be changed to named import and export.

Reply:
Yeah. I agree with this. I'll do that :)